tools/taskcluster/get-sentry-token.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script talks to the taskcluster secrets service to obtain the
+Sentry token and write it to the .sentry_token file in the root
+directory.
+"""
+
+import os
+import taskcluster
+
+# Get JSON data from taskcluster secrets service
+secrets = taskcluster.Secrets({'baseUrl': 'http://taskcluster/secrets/v1'})
+data = secrets.get('project/focus/tokens')
+
+token_file_path = os.path.join(os.path.dirname(__file__), '../../.sentry_token')
+with open(token_file_path, 'w') as token_file:
+    token_file.write(data['secret']['sentryToken'])
+
+print("Imported sentry token from secrets service")
```

Comment:
This code has been copied a thousand times. Eventually someone should just write a script that takes three arguments:
* Name of secret (project/focus/tokens)
* Name of key (sentryToken)
* File to write to (.sentry_token)

But this would be a new issue to file... :)